src/util_i_csvlists.nss

```diff
@@ -268,25 +269,23 @@ string RemoveListItem(string sList, string sListItem)
     return DeleteListItem(sList, FindListItem(sList, sListItem));
 }
 
-int CopyListItem(string sSource, string sTarget, int nIndex, int bAddUnique = FALSE, int nRange = 1)
+string CopyListItem(string sSource, string sTarget, int nIndex, int nRange = 1, int bAddUnique = FALSE)
 {
-    string sValue, sCheck;
-    int i, nCopied, nCount = CountList(sSource);
+    string sValue;
+    int i, nCount = CountList(sSource);
 
-    if (nIndex < 0 || nIndex > nCount || !nCount)
-        return FALSE;
+    if (nIndex < 0 || nIndex >= nCount || !nCount)
+        return "";

```

Comment:
If the list item is not added, the target list should be returned rather than an empty list.

Reply:
Dun.